content/news/2022-08-02-release-22.08-highlights.md

```diff
@@ -0,0 +1,138 @@
+
+Signature help provides documentation as you type the arguments to a function
+call and tracks which function parameter is currently being entered. Signature
+help is enabled by default.
+
+## LSP document highlight
+
+{{ asciinema(id="document-highlight", width="94", height="25") }}
+
+The document highlight request (`Space-h`) creates a selection for all
+instances of the symbol under the primary cursor. Helix has robust support for
+multiple selections, so you may edit all selections simultaneously (for
+example with `c`) or cycle between selections (`(`/`)`).
+
+## LSP diagnostics pickers
+
+{{ asciinema(id="diagnostic-picker", width="94", height="25") }}
+
+The new buffer and workspace diagnostics pickers may be used to jump to
+Language Server diagnostics like warnings and errors. Use `Space-g` to
+open the picker with diagnostics for the current buffer and `Space-G`
+to view all diagnostics in a workspace.
+
+## Jumplist picker
+
+{{ asciinema(id="jumplist-picker", width="94", height="25") }}
+
+The jumplist saves a history of selections. Save selections with `C-s` and jump
+forward with `C-i` and backward with `C-o`. The jumplist is a powerful tool,
+especially when working with Language Server goto-definition or global search
+which both save to the jumplist automatically.
+
+22.08 adds a new picker that can be used to jump across entries in the jumplist.
+The preview pane shows the line of the saved primary selection. Bring up the
+jumplist picker with `Space-j`.
+
+## External formatters
+
+{{ asciinema(id="external-formatter", width="94", height="25") }}
+
+Many Language Servers provide format-on-save capabilities. For languages
+without Language Servers or for Language Servers that do not implement
+formatting, an external formatter binary may now be configured. The document
+is passed through the formatter's stdin and replaced with the formatted output
+from stdout.
+
+An external formatter may also be used when you prefer the formatting from an
+external tool over formatting provided by a Language Server. For example if you
+configure `black` for formatting Python, formatting will be accomplished with
+`black` rather than `pylsp`.
+
+## Bracketed paste
+
+{{ asciinema(id="bracketed-paste", width="94", height="25") }}
+
+Bracketed paste is terminal emulator feature that allows terminal programs to
+recognize paste sequences and handle the pasted text. Without bracketed paste
+support, text pasted with operating-system level paste (`C-v`) looked to Helix
+like text that was entered very quickly which lead to some odd side-effects
+like awkward indentation or mysteriously appearing auto-pair characters.
+
+22.08 adds support for bracketed paste, so now all pastes into Helix from
+terminal emulators that support bracketed paste work as if you had pressed
+`Space-p` in normal mode. In the above cast, the entire text of _Moby-Dick_
+is pasted with `C-v` in insert mode instantly.
+
```

Comment:
Do we also want to add a note that we now have `testers-needed` label? Like https://github.com/helix-editor/helix/pull/2267 to let users try out stuff in case anyone is interested? Or a section to show those stuff?

Reply:
Eh, I'd prefer to keep the specifics of upcoming changes out of the release notes since they aren't certain to be in the release until they're merged/released. Plus I think the label probably appeals more to people in the helix repo. I think these release notes mostly appeal to people who don't follow development.